tests/unit/services/test_lifecycle.py

```diff
@@ -356,4 +364,29 @@ def test_repr(fake_parts_lifecycle, app_metadata, fake_project):
     )
 
 
+def test_post_prime_success(fake_parts_lifecycle):
+    step_info = StepInfo(
+        part_info=PartInfo(
+            fake_parts_lifecycle.project_info, part=Part("test-part", {})
+        ),
+        step=Step.PRIME,
+    )
+
+    assert not fake_parts_lifecycle.post_prime(step_info)
```

Comment:
here's a question for @cmatsuoka: how come the callbacks have to return a boolean? From what [I can tell](https://github.com/canonical/craft-parts/blob/main/craft_parts/callbacks.py#L199) that value isn't used